docs/source/user/aerodyn-olaf/InputFiles.rst

```diff
@@ -312,6 +312,118 @@ of a box of shape 5x20x30 and dimension 1200x300x295.  The grid contains both th
+   ! Advanced options may be placed here in arbitrary order, with the regular format:
+   ! Value1   Key1            - Comment1
+   ! Value2   Key2            - Comment2
+   ! Lines starting with `!` are ignored, empty lines are ignored
+   [...] etc
+
+Currently, the advanced options supported as as follows:
+
+.. code::
+
+   ===============================================================================================
+   --------------------------- ADVANCED OPTIONS --------------------------------------------------
+   ===============================================================================================
+   "Panels.vtk"  SrcPnlFile     - Name of VTK file containing source panels {default: ""}
+   1             nSrcPnlUpdate  - How often do src panel updates (in time steps of OLAF), {default: 1}
+   True          Induction      - Compute induction, {default: True} 
+   True          InductionAtCP  - Compute induced velocities at nodes or CP, {default: True} 
+   True          WakeAtTE       - Start the wake at the trailing edge, or at the LL, {default: True} 
+   False         DStallOnWake   - Dynamic stall has influence on wake, {default: False} 
+   0.75          kFrozenNWStart - Fraction of wake induced velocity at start of frozen wake, {default: 0.75} 
+   0.5           kFrozenNWEnd   - Fraction of wake induced velocity at end of frozen wake, {default: 0.5} 
+   0.0           zGround        - Ground height {default: 0.0}
+   0.1           zGroundPush    - Ground push back {default: 0.1}
+
+
+
+**SrcPanlFile** [string] specifies the name of the VTK file used for the source panel method.
+The VTK file should be a legacy ASCII VTK file, with DATASET POLYDATA POINTS and POLYGONS.
+A sample VTK file is provided below for two panels forming a regular grid in the XY plane. 
+The connectivity of the polygons needs to be such that the normal points away from the body and
+into the fluid. In the example below, the normals are in the z direction which would be typical 
+for a bottom wall with fluid above. 
+When use **WrVTK**, OLAF will write separate VTK files containing various information related to the source panels, such as the pressure coefficient, area, force per area, normals. Looking at the orientation of the normals is extremely important (In Paraview, select 3D Glyph, Arrows, and select the `Normals` output, scaled by the `Normals`). 
+The BodyID CELL_DATA can be used to separate different patches, which can help the post processing. This is optional, OLAF doesn't use it currently, but it is written back in the output files of OLAF.
+
+Curious readers may look at the unittest `Test_SrcPnl_Sphere` of OLAF that test the pressure coefficient
+about a sphere.
+
+.. code::
+
+   # vtk DataFile Version 2.0
+   Comment
+   ASCII
+   DATASET POLYDATA
+   POINTS 6 double
+   0.0 0.0 0.0
+   0.0 25.0 0.0
+   0.0 50.0 0.0
+   10.0 0.0 0.0
+   10.0 25.0 0.0
+   10.0 50.0 0.0
+
+   POLYGONS 2 10
+   4 0 1 4 3
+   4 1 2 5 4
```

Comment:
These polygons are defined clockwise when viewed from above.  Wouldn't that mean (following the right-hand rule) that the normal vector points towards -z?